docs/examples/voting.rst

```diff
@@ -176,7 +176,7 @@ of votes.
         // Calls winningProposal() function to get the index
         // of the winner contained in the proposals array and then
         // returns the name of the winner
-        function winnerName() public view
+        function winnerName() external view
                 returns (bytes32 winnerName_)
         {
             winnerName_ = proposals[winningProposal()].name;
```

Comment:
`winningProposal` is used here. So that has to be made public. Here's the CI check that failed: https://app.circleci.com/pipelines/github/ethereum/solidity/17880/workflows/1a6d2776-037e-462b-a52c-a88feed5f637/jobs/799349

Could you please fix it in this PR / branch itself? We have a section on workflow for pull requests that may be relevant here: https://docs.soliditylang.org/en/v0.8.6/contributing.html?highlight=evmone#workflow-for-pull-requests

Reply:
Ok i will use the same PR request to make new changes

Reply:
Thanks. You can ping me by using "re request" review button next to the Reviewers list.

Reply:
Could you also update other files from `docs/examples/*.rst`?

Reply:
Of course i will update the others as well